src/part_b/CapGain.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * @author snishi
+ * @date 7/10/2021
+ */
+
+package part_b;
+
+import java.util.Scanner;
+
+public class CapGain {
+    private double totCapGain;
+    private Queue sharesHeld;
+
+    public CapGain(){
+        totCapGain = 0.0;
+        sharesHeld = new Queue();
+    }
+
+    public void updateTotCapGain(double totCapGain){
+        this.totCapGain += totCapGain;
+    }
+
+    public double getTotCapGain(){
+        return totCapGain;
+    }
+
+    /**
+     * @about prompt for main menu items
+     */
+    public static void mainMenuPrompt(){
+        System.out.println("********Main Menu********");
+        System.out.println("1. Buy");
+        System.out.println("2. Sell");
+        System.out.println("3. Total Capital Gain So Far");
+        System.out.println("4. Quit");
+    }
+
+    /**
+     * @about add numShares shares at price price to the queue
+     * @param numShares number of shares bought
+     * @param price price at which shares are bought
+     */
+    public void buy(int numShares, double price){
+        sharesHeld.enqueue(numShares, price);
+    }
+
+    /**
+     * @about sell numShares shares at price price, dequeue
+     * @param numShares number of shares sold
+     * @param price price at which shares are sold
+     */
+    public void sell(int numShares, double price){
+        if(sharesHeld.isEmpty()){
+            System.out.println("ERROR: cannot sell when you havent bought anything");
```

Comment:
i suggest using a logger function as a wrapper to add "ERROR:" for consistency and ease of changing

Reply:
seems to break the consistency of other comments. do all errors start with ERROR? do other errors all have logs?